platform/util/platform-util/src/main/java/org/codice/ddf/platform/util/XMLUtils.java

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.platform.util;
+
+import java.io.StringWriter;
+import java.io.Writer;
+
+import javax.xml.transform.Source;
+import javax.xml.transform.Transformer;
+import javax.xml.transform.TransformerException;
+import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.dom.DOMSource;
+import javax.xml.transform.stream.StreamResult;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.w3c.dom.Node;
+
+/**
+ * Utility for handling XML
+ */
+public class XMLUtils {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(XMLUtils.class);
+
+    /**
+     * Transforms XML into a String
+     *
+     * @param sourceXml           to transform a given DOMSource
+     * @param transformProperties settings for transformer
+     * @return XML string
+     */
+    public static String transformXml(Source sourceXml,
```

Comment:
You could also split this method into multiple methods.  One that returns a Source or Node could be `transforms` and the method that returns a string could be `format` for example.

I would not include the word XML in your method names since the class is XMLUtils.